packages/react-heartwood-components/src/components/Icon/Icon.js

```diff
@@ -21,20 +21,24 @@ export type Props = {
 const Icon = (props: Props) => {
 	const { icon, customIcon, isLineIcon, className, ...rest } = props
 
-	if (
-		!customIcon &&
-		(!icon || !icon.toLowerCase || !icons[icon.toLowerCase()])
-	) {
+	const iconKey = icon && icon.toLowerCase && icon.toLowerCase()
```

Comment:
`typeof icon === 'string' && icon.toLowerCase()`, since stringiness is what you're trying to establish